<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addToNovo</class>
 <widget class="QDialog" name="addToNovo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="origemGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Origem</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="producaoInternaCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Producao Interna</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="dataNascimentoLabel">
     <property name="text">
      <string>Data de Nascimento:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dataNascimentoDateTimeEdit"/>
   </widget>
   <widget class="QCheckBox" name="compradoCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprado</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="dataCompraLabel">
     <property name="text">
      <string>Data de Compra:</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dataCompraDateTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <zorder>producaoInternaCheckBox</zorder>
   <zorder>splitter_5</zorder>
   <zorder>splitter_4</zorder>
   <zorder>compradoCheckBox</zorder>
   <zorder>splitter_4</zorder>
   <zorder>producaoInternaCheckBox</zorder>
   <zorder>splitter_5</zorder>
   <zorder>compradoCheckBox</zorder>
  </widget>
  <widget class="QPushButton" name="adicionarPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="limparPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelarPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>125</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="categoriaLabel">
       <property name="text">
        <string>Categoria:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="categoriaComboBox">
       <item>
        <property name="text">
         <string>Boi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vaca</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Touro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vitelo</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="sexoLabel">
       <property name="text">
        <string>Sexo: </string>
       </property>
      </widget>
      <widget class="QComboBox" name="sexoComboBox">
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="totalLabel">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="totalComboBox"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
